### DISCUSSION TITLE
[perfromance][chat] Optimize chat subscriptions

### DISCUSSION THREAD

**Discussion by flexsurfer** [status-im/status-legacy#16618] [General]
Currently, we have lots of computations and business login inside chat subscriptions, we should move all that into events , we should have only one subscription where we get messages stream from app-db

![image](https://user-images.githubusercontent.com/11790366/79863740-8d98d700-83d8-11ea-9dea-7294aefa9d0c.png)

![image](https://user-images.githubusercontent.com/11790366/79863801-ab663c00-83d8-11ea-8841-a1e5bea30122.png)

**Comment by churik** [comment]
@flexsurfer 
is the issue still relevant according to the new design implementation?

**Comment by flexsurfer** [comment]
yes

**Comment by cammellos** [comment]
Moving to discussion for now, and we can re-open when ready to be worked on
